cfme/tests/networks/nuage/test_operations.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from cfme.utils.log import logger
+from cfme.utils.wait import wait_for
+from wrapanapi.utils.random import random_name
+from cfme.networks.provider.nuage import NuageProvider
+
+pytestmark = [
+    pytest.mark.provider([NuageProvider])
+]
+
+
+def test_router_add_subnet(provider, with_nuage_sandbox):
+    """
+    Ensure that subnet is added on network router
+
+    We navigate to router through Provider > Tenant > Network Router
+    """
+    sandbox = with_nuage_sandbox
+    tenant_name = sandbox['enterprise'].name
+    router_name = sandbox['domain'].name
+    tenant = provider.collections.cloud_tenants.instantiate(name=tenant_name, provider=provider)
+    router = tenant.collections.routers.instantiate(name=router_name)
+    subnet_name = random_name()
+    router.add_subnet(subnet_name, '100.100.100.0', '255.255.255.0', '100.100.100.1')
```

Comment:
Q: are you sure about such hard-coded value?

Reply:
It's just some random IPs, so the popup is filled with this data and we can check if the subnet was really created.

Reply:
other than getway... I'm able to get random data.but thanks for answer.

Reply:
Please use a non-routable network (10.0.0.0/24, 172.16.0.0/16, 192.168.0.0/16) unless the UI blocks these from being added.

Reply:
This change hasn't been made yet, but the PR looks merge-ready otherwise - @aljazkosir Please confirm whether or not you can change this subnet IP range.

Reply:
Ops, I though I pushed this, it appears I only commited it though..